tests/conftest.py

```diff
@@ -1,5 +1,13 @@
 import pytest
-from brownie import config, Wei, Contract
+from brownie import config, Contract, convert, config
+
+
+@pytest.fixture(scope="module")
+def Strategy():
+    Strategy = config["strategy"]["name"]
+    Strategy = getattr(__import__("brownie"), Strategy)
+    yield Strategy
```

Comment:
Didn't `from brownie import Strategy` work?

Reply:
that only works for when you strategy sits in Strategy.sol . In this case, the strategy is in StrategyCurveTwoPool.sol At this point you can probably tell that I really like centralizing hard-coded variables in brownie-config.yml. This fixture reads the strategy name from the config file and loads the corresponding class during test time.